server/start.go

```diff
@@ -247,7 +247,9 @@ func startInProcess(ctx *Context, clientCtx client.Context, appCreator types.App
 
 	config := config.GetConfig(ctx.Viper)
 	if err := config.ValidateBasic(); err != nil {
-		return err
+		ctx.Logger.Info("The minimum-gas-prices config in app.toml is set to the empty string. "+
```

Comment:
Can we do `Logger.Error` or will that cause a panic?

Reply:
```suggestion
		ctx.Logger.Info("WARNING: The minimum-gas-prices config in app.toml is set to the empty string. "+
```

Reply:
Switched to error:

<img width="1009" alt="Screenshot 2021-07-02 at 17 17 20" src="https://user-images.githubusercontent.com/1293565/124295726-7798b180-db59-11eb-8f8a-1b4e7df694e3.png">

Too bad the logger doesn't have a .Warn that's yellow, that would be the most accurate